arbnode/transaction_streamer.go

```diff
@@ -1928,35 +1937,41 @@ func (s *TransactionStreamer) submitTransactionsToEspresso(ctx context.Context,
 			return retryRate
 		}
 	}
-	return s.espressoTxnsPollingInterval
+	return s.espressoTxnsSendingInterval
 }
 
+// This method aquires the submitted txn mutex lock. This wil cause the thread tht polls for finality and this thread to fight over the mutex.
```

Comment:
1) Why exactly are we also building batches here? Do we still need the `shouldResubmitEspressoTransactions` function if we are not calling it here anymore?
2) Very minor nit: tht -> that